python/tvm/relay/grammar/Relay.g4

```diff
@@ -131,7 +132,7 @@ identType: CNAME ;
 // Float16, Float32, Float64
 // Bool
 
-body: '{' expr '}' ;
+body: '{' expr ';' '}' ;
```

Comment:
This syntax change my deserve a bit of discussion. We could add it, have it be optional, or not add it at all. There original justification for not having it is that `;` was a logical replacement for `in` for let expressions, separating it from the imperative use of semicolons. To me, imperative uses of semicolons serve to separate individual statements, whereas the functional use here indicates chaining. It also opens up the question of whether to require/allow/forbid semicolons between global definitions.

Reply:
I'm a fan of using semi-colons everywhere because it makes parsing simpler. The text format should be relatively straight forward to parse.

Reply:
I don't think it should go in this PR. It needs to be added in several places and test cases need to be updated. Could you open an issue?